apps/web/src/components/doc-editor/docSheet.css

```
@@ -0,0 +1,274 @@
+  background: var(--doc-sheet-bg);
+  color: var(--doc-sheet-text);
+  border: 1px solid var(--color-border);
+  box-shadow: 0 1px 12px var(--color-shadow);
+  container-type: inline-size;
+  /*
+   * One typographic point of the document, as a share of the page's width.
+   * A4 is 595.28pt wide, so 1pt is 100/595.28 container-query width units.
+   * Stored point sizes (the avatar, its border) render against this.
+   */
+  --doc-sheet-pt: 0.168cqw;
+  font-family: var(--font-body);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)